Sources/Orbit/Support/Components/TimelineStepContent.swift

```diff
@@ -0,0 +1,34 @@
+import SwiftUI
+
+public struct TimelineStepBadgeText: View {
+
+    let label: String
+    let sublabel: String
```

Comment:
can have default values

Reply:
Let's omit it here as it will require explicit init declaration